### ISSUE TITLE
bitnami-compat/rabbitmq:3.9.23 failing to start up on `linux/amd64` macOS

### ISSUE THREAD

**Issue by nediamond** [ZCube/bitnami-compat#35]
I have confirmed that this issue is not present in `3.9.22` by switching versions and starting them up repeatedly

I have it defined in a docker compose file like this:
```
  rabbitmq:
    # https://github.com/zcube/bitnami-compat
    image: ghcr.io/zcube/bitnami-compat/rabbitmq:3.9.23
    volumes:
      - local_rabbitmq_data:/bitnami
    ports:
      - "15672:15672"
```

and when bringing it up it fails with these logs:
```
-rabbitmq-1  | rabbitmq 20:15:18.71 
-rabbitmq-1  | rabbitmq 20:15:18.71 Welcome to the Bitnami rabbitmq container
-rabbitmq-1  | rabbitmq 20:15:18.71 Subscribe to project updates by watching https://github.com/bitnami/containers
-rabbitmq-1  | rabbitmq 20:15:18.71 Submit issues and feature requests at https://github.com/bitnami/containers/issues
-rabbitmq-1  | rabbitmq 20:15:18.71 
-rabbitmq-1  | rabbitmq 20:15:18.72 INFO  ==> ** Starting RabbitMQ setup **
-rabbitmq-1  | rabbitmq 20:15:18.73 INFO  ==> Validating settings in RABBITMQ_* env vars..
-rabbitmq-1  | rabbitmq 20:15:18.74 INFO  ==> Initializing RabbitMQ...
-rabbitmq-1  | rabbitmq 20:15:18.75 INFO  ==> Generating random cookie
-rabbitmq-1  | rabbitmq 20:15:18.77 INFO  ==> Starting RabbitMQ in background...
-rabbitmq-1  | rabbitmq 20:17:00.86 ERROR ==> Couldn't start RabbitMQ in background.
-rabbitmq-1 exited with code 1
```

**Comment by nediamond** [comment]
sorry just realized this issue is probably not caused by this repo! closing

**Comment by ZCube** [comment]
@nediamond fixed at 16abf881f914270f6612c0684adecd6835388ac6
